generator/src/main/java/com/scottlogic/deg/generator/generation/fieldvaluesources/DateTime/RandomDateIterator.java

```diff
@@ -0,0 +1,35 @@
+package com.scottlogic.deg.generator.generation.fieldvaluesources.DateTime;
+
+import com.scottlogic.deg.generator.utils.RandomNumberGenerator;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.Iterator;
+
+public class RandomDateIterator implements Iterator<LocalDateTime> {
```

Comment:
Can we get some unit tests for these two iterators?

Reply:
Unit tested as part of TemporalFieldValueSourceTests